zstacklib/zstacklib/storage/drbd/resource.py

```
@@ -0,0 +1,429 @@
+        self.remote_role = None  # type: Optional[str]
+        self.local_disk_state = None  # type: Optional[str]
+        self.remote_disk_state = None  # type: Optional[str]
+        self.exists = False
+        
+        if self.name is None:
+            return
+        
+        try:
+            self._init_from_name()
+            self.exists = True
+        except Exception:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)